lib/dma.h

```diff
@@ -386,6 +386,16 @@ int
 dma_controller_dirty_page_get(dma_controller_t *dma, vfu_dma_addr_t addr,
                               uint64_t len, size_t pgsize, size_t size,
                               char *bitmap);
+
+void
```

Comment:
if this is for the test, can you put the tests at bottom of dma.c in an #ifdef TEST, and compile that?

Reply:
This is all covered in the tests for the VFIO_DEVICE_FEATURE_DMA_LOGGING_REPORT tests so we don't call these functions directly in the tests - I'll make them `static`.